pom.xml

```diff
@@ -11,14 +11,15 @@
     <description>CaseService</description>
 
     <properties>
-        <springboot.version>2.7.18</springboot.version>
+        <springboot.version>3.3.3</springboot.version>
         <common.version>10.49.4</common.version>
         <jackson.version>2.16.1</jackson.version>
-        <java.version>17</java.version>
+        <java.version>21</java.version>
         <surefire.version>3.2.5</surefire.version>
-        <godaddylogging.version>1.2.5</godaddylogging.version>
+        <!-- godaddylogging.version>1.2.5</godaddylogging.version -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <failsafe.skipITs>true</failsafe.skipITs>
```

Comment:
To allow me to control the tests during development. This will of course be `false` when delivered.